stable/kube-bench/values.yaml

```diff
@@ -0,0 +1,11 @@
+provider: eks
+
+cronjob:
+  schedule: "*/1 * * * *"
```

Comment:
I think every minute is too frequent?? 😅

Reply:
yes it is. Making it monthly.